gcp_variant_transforms/testing/data/sharding_configs/integration_with_residual.yaml

```diff
@@ -60,6 +60,11 @@
        - "chrM"
        - "m"
      total_base_pairs: 57,227,415
+-  output_table:
+     table_name_suffix: "empty"
+     regions:
+       - "does not match to any reference_name value"
```

Comment:
How does this comment work? Isn't region part going to be looking for it?

Reply:
Modified to make it more clear I meant empty shard.